DataServer/email_sender.py

```diff
@@ -0,0 +1,59 @@
+import smtplib
+from email.message import EmailMessage
+class email_sender:
+    
+    def __init__(self, logger, subject=None, body=None, configDict=None) -> None:
+        if configDict is not None:
+            self.username = configDict['email_username']
+            self.password = configDict['email_password']
+            self.send_to = configDict['eamil_receiver']
+            self.subject = configDict['email_subject']
+            self.body = configDict['email_content']
+        else: 
+            raise RuntimeError('Need to pass the configDict to instantiate the email')
+        if subject is not None: 
+            self.subject = subject
+            
+        if body is not None:
+            self.body = body
```

Comment:
Why are we overriding these values? This should be the job of the caller to determine the appropriate values.